lib/tree_stand/node.rb

```diff
@@ -11,15 +11,57 @@ class Node
     extend Forwardable
     include Enumerable
 
+    # @!method changed?
+    #   @return [Boolean] true if a syntax node has been edited.
+    # @!method child_count
+    #   @return [Integer] the number of children nodes.
+    # @!method extra?
+    #   @return [Boolean] true if the node is *extra* (e.g. comments).
+    # @!method has_error?
+    #   @return [Boolean] true if the node is a syntax error or contains any syntax errors.
+    # @!method missing?
+    #   @return [Boolean] true if the parser inserted that node to recover from error.
+    # @!method named?
+    #   @return [Boolean] true if the node is not a literal in the grammar.
+    # @!method named_child_count
+    #   @return [Integer] the number of *named* children.
     # @!method type
     #   @return [Symbol] the type of the node in the tree-sitter grammar.
     # @!method error?
     #   @return [bool] true if the node is an error node.
-    def_delegators(
-      :@ts_node,
-      :type,
-      :error?,
-    )
+    def_delegators(*
+      %i[
+        @ts_node
+        changed?
+        child_count
+        error?
+        extra?
+        has_error?
+        missing?
+        named?
+        named_child_count
+        type
+      ])
+
+    # These are methods defined in {TreeStand::Node} but map to something know
+    # in {TreeSitter::Node}, because we want a more idiomatic API.
```

Comment:
```suggestion
    # These are methods defined in {TreeStand::Node} but map to something in
    # {TreeSitter::Node}, because we want a more idiomatic API.
```